### PR TITLE
Reorganize the federation PR builder config to allow bringing only federation control plane up and down.

### PR DESCRIPTION
**Pull request by madhusudancs**
This PR depends on #1957 which is being reviewed independently. Please review only the last commit here.

### CODE REVIEW

File: jobs/pull-kubernetes-federation-e2e-gce.sh [hunk 1]
```diff
@@ -20,52 +20,45 @@ set -o xtrace
...
-fi
-
 export FEDERATION="true"
-export PROJECT="${PROJECT:-k8s-jkns-pr-bldr-e2e-gce-fdrtn}"
-export FEDERATION_PUSH_REPO_BASE="gcr.io/k8s-jkns-pr-bldr-e2e-gce-fdrtn"
-export GINKGO_PARALLEL="n" # We don't have namespaces yet in federation apiserver, so we need to serialize
-export GINKGO_TEST_ARGS="--ginkgo.focus=\[Feature:Federation\]"
-export E2E_ZONES="us-central1-a us-central1-f" # Where the clusters will be created. Federation components are now deployed to the last one.
-export KUBE_GCE_ZONE="us-central1-f" #TODO(colhom): This should be generalized out to plural case
+export PROJECT="${PROJECT:--k8s-jkns-pr-bldr-e2e-gce-fdrtn}"
```
Comment (shashidharatd) [inline jobs/pull-kubernetes-federation-e2e-gce.sh]: is it intentional to use double hyphen?
PROJECT will be set to "-k8s-jkns-pr-bldr-e2e-gce-fdrtn" if not defined earlier.

**Comment by shashidharatd** [comment]
/lgtm
have not seen a PR CI job deep enough though.

File: jobs/pull-kubernetes-federation-e2e-gce.sh (hunk 1 above)
Comment (madhusudancs) [inline jobs/pull-kubernetes-federation-e2e-gce.sh]: Sorry, this was by accident, not intentional. I have removed the hyphen now.

**Comment by madhusudancs** [comment]
@shashidharatd I am learning too. Also, there is no easy way to test these. So we are probably going to make changes as we test this on the PRs.

**Pull request merged**